src/pokemon-client.js

```diff
@@ -0,0 +1,36 @@
+class PokemonClient {
+   constructor() {
+      this.API_URL = "https://pokeapi.co/api/v2/pokemon/";
+   }
+
+   async getPokemon(pokemonID) {
+      try {
+         const response = await fetch(this.API_URL + pokemonID);
+         const data = await response.json();
+
+         return data;
+      } catch (error) {
+         return false;
+      }
+   }
+
+   async getAllPokemonNames() {
+      try {
+         const numberOfPokemons = 898;
```

Comment:
what is this magic number?

Reply:
It is the number of Pokemons that exist:)
Maybe it will be better to get an API request that get this number. I'll have to check again where I found it